bidwire/scrapers/knoxville_tn_meetings_scraper.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import scrapelib
+from lxml import html
+
+from sqlalchemy.exc import IntegrityError
+
+from document import Document
+from scrapers.base_scraper import BaseScraper
+from utils import ensure_absolute_url
+
+log = logging.getLogger(__name__)
+
+
+class KnoxvilleTNMeetingScraper(BaseScraper):
+    SITE_ROOT_URL = 'https://knoxvillecitytn.iqm2.com/Citizens/'
+    MEETING_SCHEDULE_URL = 'https://knoxvillecitytn.iqm2.com/Citizens/Calendar.aspx'
+
+    def __init__(self) -> None:
```

Comment:
It's a function annotation that indicates the return type. Mostly only used for static analysis by IDEs and such. PyCharm started adding them by default. [PEP-3107](https://www.python.org/dev/peps/pep-3107/)